operator/pkg/controllers/configconnector/configconnector_controller.go

```diff
@@ -691,6 +691,9 @@ func (r *ConfigConnectorReconciler) fetchAndApplyAllWebhookConfigurationCustomiz
 
 // applyValidatingWebhookConfigurationCustomizationCR applies customizations specified in a ValidatingWebhookConfigurationCustomization CR.
 func (r *ConfigConnectorReconciler) applyValidatingWebhookConfigurationCustomizationCR(ctx context.Context, cr *customizev1alpha1.ValidatingWebhookConfigurationCustomization) error {
+	if err := checkForDuplicateWebhooks(cr.Spec.Webhooks); err != nil {
```

Comment:
Nit: we may want to use the "apimachinery" validation libraries - gathering all the errors.  It means the errors are presented more consistently.  Example PR that makes this change in another project: https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3026/files#diff-93dbac8d674a0494b5f802a4b57b0a42b92e25196b8b31c2fc54db8ceaf22562

Reply:
Good to know! Thank you! Although for this case there is only one error. If there are duplicated webhooks in the CR, I don't think we should continue to apply it. So the code returns on the first error. Do you suggest that we should use `field.ErrorList{}` even if there is only one error?